internal/datastore/postgres/common/errors.go

```diff
@@ -19,6 +19,7 @@ const (
 	pgReadOnlyTransaction       = "25006"
 	pgQueryCanceled             = "57014"
 	pgInvalidArgument           = "22023"
+	pgMissingTable              = "42P01"
```

Comment:
there are two constants defined elsewhere: `postgresMissingTableErrorCode`

please remove them in favor of this one